Lib/test/test_gzip.py

```diff
@@ -359,6 +360,13 @@ def test_read_with_extra(self):
         with gzip.GzipFile(fileobj=io.BytesIO(gzdata)) as f:
             self.assertEqual(f.read(), b'Test')
 
+    def test_fileobj_without_name(self):
```

Comment:
I suggest to backport `test_fileobj_from_fdopen` from 3.x.

And add a similar test with `fileobj=io.open(fd, 'wb')` instead of `fileobj=os.fdopen(fd, 'wb')`.

Reply:
> I suggest to backport test_fileobj_from_fdopen from 3.x.

I'm not sure I follow. The only difference between 2.7's `test_fileobj_from_fdopen` and 3.7's is the assert:
[2.7](https://github.com/python/cpython/blob/2.7/Lib/test/test_gzip.py#L332)
[3.7](https://github.com/python/cpython/blob/3.7/Lib/test/test_gzip.py#L432)

Could you clarify?

Reply:
All right then.